dynamo-formatters/src/test/scala/uk/gov/nationalarchives/dynamoformatters/DynamoFormattersTest.scala

```diff
@@ -317,6 +328,20 @@ class DynamoFormattersTest extends AnyFlatSpec with TableDrivenPropertyChecks wi
     assetRowInvalidValue.ingestedPreservica should equal(false)
   }
 
+  "assetTableFormat read" should "return the value if skipIngest is present and false otherwise" in {
+    val skipIngestPresentFalse = assetTableFormat.read(buildAttributeValue(allAssetFieldsPopulated)).value
+    skipIngestPresentFalse.skipIngest should equal(false)
+
+    val skipIngestPresentTrue =
+      assetTableFormat.read(buildAttributeValue(allAssetFieldsPopulated + (skipIngest -> fromBool(true)))).value
+
+    skipIngestPresentTrue.skipIngest should equal(true)
+
+    val skipIngestMissing =
+      assetTableFormat.read(buildAttributeValue(allAssetFieldsPopulated.filter(_._1 != skipIngest))).value
```

Comment:
Maybe `.filter{ case(field, _) => field != skipIngest}` instead?

Reply:
That's nicer, I've changed it.